src/containers/Navbar/Navbar.container.tsx

```
@@ -0,0 +1,89 @@
+                                onSubmit={handleSubmit}
+                            />
+                        </Stack>
+                    )}
+                    <Stack
+                        gap={2}
+                        flexDirection={'row'}
+                        alignItems={'center'}
+                        justifyContent={'center'}
+                    >
+                        <StyledShadowedLink to="/notification">
+                            <Notifications></Notifications>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)